base/applications/cmdutils/taskkill/taskkill.c

```diff
@@ -440,89 +463,137 @@ static BOOL add_to_task_list(WCHAR *name)
     return TRUE;
 }
 
-/* FIXME Argument processing does not match behavior observed on Windows.
- * Stringent argument counting and processing is performed, and unrecognized
- * options are detected as parameters when placed after options that accept one. */
+static int is_valid_arguments(WCHAR *argument)
```

Comment:
I think this name does not fully represent what this function does

Reply:
You are right, thank you for pointing out, I will fix it soon.